src/components/GameView/GameUnavailableOverlay.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <v-overlay
+    data-cy="unavailable-game-overlay"
+    id="unavailable-game-scrim"
+    v-model="unavailableGame"
+    class="game-overlay text-center d-flex justify-center align-center"
+  >
+    <div class="overlay-header">
+      <h1 :class="[this.$vuetify.display.xs === true ? 'text-h5' : 'text-h3', ['font-weight-bold']]">
+        Oops!
+      </h1>
+      <p>Looks like this game isn't available</p>
+    </div>
+    <v-btn
+      color="secondary"
+      class="mt-4"
+      data-cy="leave-unavailable-game-button"
+      :loading="leavingGame"
+      @click="this.$router.push('/')"
+    >
+      Go Home
+    </v-btn>
+  </v-overlay>
+</template>
+
+<script>
+export default {
+  name: 'GameUnavailable',
+  props: {
+    unavailableGame: {
```

Comment:
Let's rename this `show`
```suggestion
    show: {
```